tracex_project/db_results/forms.py

```diff
@@ -25,3 +29,77 @@ def get_patient_journey_choices(self):
         choices = [(pj.name, pj.name) for pj in patient_journeys]
 
         return choices
+
+
+class EvaluationForm(BaseEventForm):
+    """Form for evaluating the extraction result."""
+
+    min_age = forms.IntegerField(
+        required=False,
+        widget=forms.NumberInput(attrs={"id": "min-age"}),
+    )
+    max_age = forms.IntegerField(
+        required=False,
+        widget=forms.NumberInput(attrs={"id": "max-age"}),
+    )
+    none_age = forms.BooleanField(
+        label=mark_safe("<i>Include elements with None values</i>"),
+        required=False,
+        widget=forms.CheckboxInput(),
+    )
+    gender = forms.MultipleChoiceField(
+        label="Gender:",
+        choices=[],
+        widget=forms.CheckboxSelectMultiple(),
+        required=False,
+    )
```

Comment:
Based on what we want to display, we should consider using "sex", as it's more relevant in the health domain

Reply:
In terms of biological sex i agree with you, but if a patient states in the journey that he identifies as non-binary or something similar "sex" would be the wrong category.

Reply:
Thats true, but thats why we should think about, what we want to display. And we should (or more likely could) adjust the prompts accordingly